content/applications/marketing/marketing_automation/getting_started/target_audience.rst

```diff
@@ -0,0 +1,119 @@
+==================
+Target an audience
+==================
+
+Delivering marketing campaigns to the *right* audience is paramount when trying to grow a business.
+The :guilabel:`Odoo Marketing Automation` application helps marketers to do just that by providing
+filtering tools, that can be as simple or as complex as necessary, to reach the right customers at
+the right time.
+
+Work with target filters
+========================
+
+When configuring the filters on a marketing campaign, there are some options that have an arrow 
+icon beside them. The arrow signifies that the filter has more refined parameters within it. 
+
+.. image:: target_audience/marketing-filters.png
+   :align: center
+   :alt: The drop-down filter menu in the Marketing Automation application.
+
+Filters can be extended by adding *branches* and *nodes*. A *branch* creates a new line of
+parameters, and further allows parameters to be clustered with *Any* or *All* statements, and a
+*node* adds another parameter to the branch.
+
+Every time a new branch is created, there are two options:
+
+- Either the records can match :guilabel:`ALL` criteria for the upcoming rules (an AND statement
+  where all criteria must match).
+
+- Or, the records can match :guilabel:`ANY` criteria for the upcoming rules (an OR statement where
+  only one of the criteria must match).
+
+To change between these two options, simply click the drop-down arrow icon in the green box and
+select :guilabel:`Any` or :guilabel:`All`.
+
+To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another branch click on the
+:guilabel:`... (ellipses icon)`. To exclude a :guilabel:`node` (or a :guilabel:`branch`), click on
+:guilabel:`x (delete)`.
+
+.. image:: target_audience/marketing-filter-nodes.png
+   :align: center
+   :alt: The drop-down filter menu in the Marketing Automation application.
+
+Use cases
+=========
+
+The following scenarios are examples of some filters a marketing campaign might use.
+
+**Scenario**: Narrow the target down to new opportunities in the pipeline.
```

Comment:
Think this would better serve as an H3 to differentiate the scenarios and break up the text.
```suggestion
Scenario #1: Narrow the target down to new opportunities in the pipeline.
-------------------------------------------------------------------------
```